src/SAWScript/Crucible/MIR/Override.hs

```diff
@@ -226,8 +239,16 @@ learnPointsTo ::
   MS.PrePost                 ->
   MirPointsTo                ->
   OverrideMatcher MIR w ()
-learnPointsTo _opts _sc _cc _spec _prepost _pt =
-  panic "learnPointsTo" ["not yet implemented"]
+learnPointsTo opts sc cc spec prepost (MirPointsTo md allocIdx referents) =
+  mccWithBackend cc $ \bak ->
+  do let col = cc ^. mccRustModule . Mir.rmCS ^. Mir.collection
+     globals <- OM (use overrideGlobals)
+     Some mp <- resolveAllocIndexMIR allocIdx
+     let mpMirTy = mp^.mpMirType
+     let mpTpr = tyToShapeEq col mpMirTy (mp^.mpType)
+     val <- firstPointsToReferent referents
+     v <- liftIO $ Mir.readMirRefIO bak globals Mir.mirIntrinsicTypes (mp^.mpType) (mp^.mpRef)
```

Comment:
The final place in the patch that makes use of the `crucible-mir` memory model, this time to read from a reference.